sdk_container/src/third_party/coreos-overlay/app-admin/sdnotify-proxy/sdnotify-proxy-9999.ebuild

```
@@ -2,17 +2,15 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-CROS_WORKON_PROJECT="kinvolk/sdnotify-proxy"
-CROS_WORKON_LOCALNAME="sdnotify-proxy"
-CROS_WORKON_REPO="https://github.com"
+EGIT_REPO_URI="https://github.com/kinvolk/sdnotify-proxy.git"
```

It's in flatcar org now:
```suggestion
EGIT_REPO_URI="https://github.com/flatcar/sdnotify-proxy.git"
```

---

I'll update the HOMEPAGE too.